Modify text to better explain differential coding as combination of input and coded symbol

sei-jgwohlbier opened the pull request:
Modify text to better explain differential coding as combination of input and coded symbol

Make the inline example the same as the one in the included image.

Remove some trailing white space.

Commit:
Modify text to better explain differential coding as combination of input
and coded symbol.

Make the inline example the same as the one in the included image.

Remove some trailing white space.

Commit:
added more content

777arc commented:
Awesome edits!  I got a little carried away and added a ton more to that subsection, would love your review

Commit:
Change emphasis from "previous" to "output." Adresses the original point
of confusion, and makes it consistent with emphasis usage in the section.

Trailing white space removal.

sei-jgwohlbier commented:
Looks good to me. Made one change of emphasis commit.
